project/src/components/common/sorting/sorting.tsx

```diff
@@ -0,0 +1,45 @@
+import {SortingTypes} from '../../../const/sorting';
+import {useAppDispatch, useAppSelector} from '../../../hooks/';
+import classNames from 'classnames';
+import {setSortType} from '../../../store/action';
+
+
+function Sorting() {
```

Comment:
```suggestion
import {setSortType} from '../../../store/action';

function Sorting() {
```

а также порядок в импортах наведи, сначала библиотеки, потом уже остальное